catalog-rest-service/src/main/resources/json/schema/entity/services/databaseService.json

```diff
@@ -123,6 +125,12 @@
         },
         {
           "name": "Datalake"
+        },
+        {
+          "name": "Atlas"
```

Comment:
IMO we should remove Atlas and Amundsen from DB services. The output service should be created manually before running the process. We should add that in the connector docs